charts/lighthouse/templates/webhooks-ingress.yaml

```diff
@@ -1,5 +1,14 @@
+{{- $apiAll := .Capabilities.APIVersions | toStrings -}}
```

Comment:
Can’t we use .Capabilities.APIVersions.Has here ?

Reply:
testing

Reply:
It works here but it doesn't below inside the backend.  Trying a few things to see if I can get it to accept it

```bash
<.Capabilities.APIVersions.Has>: can't evaluate field Capabilities in type interface {}
```

Reply:
Yeah, you need to prefix with `$` because you're in a range.

Reply:
Updated